.github/workflows/build_deploy_onboard.yml

```diff
@@ -0,0 +1,108 @@
+name: Build, Deploy and Onboard Cloud Operator To Live Services
+
+on:
+    push:
+      branches:
+        - main
+jobs:
+  test-deploy-onboard-live:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+  
+      - name: Build and push Docker image
+        id: build-and-push
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64
+          build-args: |
+            VERSION=${{ github.sha }}
+          tags: |
+            ghcr.io/${{ github.repository }}:${{ github.sha }}
```

Comment:
It would be better to use the `docker/metadata-action` to determine the tags:
https://github.com/docker/metadata-action
In case this is executed in the context of a PR, it generates a tag like `pr-96`, which would make it much easier to find and to distinguish from production image tags.

OK to do that later if you want.

Reply:
Feel free to close if you prefer to move to that in another PR.